hummingbot/connector/exchange/ftx/ftx_exchange.pyx

```diff
@@ -81,6 +81,7 @@ cdef class FtxExchange(ExchangeBase):
     API_CALL_TIMEOUT = 10.0
     UPDATE_ORDERS_INTERVAL = 10.0
     ORDER_NOT_EXIST_CONFIRMATION_COUNT = 3
+    REFERRAL_PROGRAM = "hummingbot1"
```

Comment:
@Ed-Latoken please check the PR base. Currently it includes changes to files that should not be modified by this PR (like this FTX connector file)